### PR TITLE
Moved Appliance topic page to new layout

### PR DESCRIPTION
**Pull request by nkohari**
Implementation of mockup: https://marvelapp.com/2d1d81e/screen/15110500

Note: This requires the style information added in https://github.com/auth0/auth0-docs/issues/1081.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mpaktiti** [comment]
Hello @nkohari! I can see that [this PR](https://github.com/auth0/auth0-docs/pull/1090) is merged, so this one is good to go as well, right?

**Comment by nkohari** [comment]
Yep! It should be good to go.

File: articles/appliance/index.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/infrastructure">Appliance Infrastructure Requirements</a>
+    <p>
+      This document contains information about the Appliance and its infrastructure requirements.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/appliance-sprint">Appliance Sprint (Customer Onboarding)</a>
```
Comment (mpaktiti) [inline articles/appliance/index.html]: @nkohari The URL does not work for me. I can see that the doc is available at 
https://auth0-docs-content-pr-2316.herokuapp.com/docs/onboarding/appliance-sprint so we should either update the link to `/onboarding/appliance-sprint` or add a redirect. Not sure which is preferable.

Comment (nkohari) [reply articles/appliance/index.html]: Ah, sorry about that! The URL is wrong. I will update the PR. Good catch!

File: articles/appliance/index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    <i class="icon icon-budicon-715"></i><a href="/appliance/dashboard">Appliance Dashboard/Configuration Area</a>
+    <p>
+      Regardless of whether you choose to use Auth0's cloud services or to run Auth0 through Appliance instances, the Management Dashboard is the interface through which you may configure things like your Applications, Connections, Users, and Rules.
+    </p>
+    <ul>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/custom-domains">Custom Domains</a>
+      </li>
+      <li>
+        <i class="icon icon-budicon-695"></i><a href="/appliance/clock">Time Synchronization in the Appliance</a>
```
Comment (mpaktiti) [inline articles/appliance/index.html:88]: @nkohari For these two (Custom Domains and Time Sync) I cannot see anything highlighted on the left menu. Maybe it's expected, I just thought I let you know.

image

Comment (nkohari) [reply articles/appliance/index.html:88]: I believe that is a known issue (or intentional) but @ntotten would have to say for sure.

File: articles/appliance/index.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/modules">Node.js Modules Available in Rules and Custom Database Connections for Appliances</a>
+    <p>
+      For security reasons, rules and custom database connections for Auth0 appliances run in a JavaScript sandbox. You can use the full power of the ECMAScript 5 language and a few selected libraries.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/webtasks">Webtasks</a>
+    <p>
+      Beginning with Build 7247, you may use the Appliance's version of  [Webtasks](http://webtask.io/) to execute your rules and custom database logic.
```
Comment (mpaktiti) [inline articles/appliance/index.html]: @nkohari The markdown URL needs fixing, it does not render correctly. 

image

Comment (nkohari) [reply articles/appliance/index.html]: Another good catch. :)

File: articles/appliance/index.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/onboarding/appliance-sprint">Appliance Sprint (Customer Onboarding)</a>
+    <p>
+      Appliance Sprint is Auth0’s onboarding program for enterprise customers choosing an Auth0 Appliance. It helps you achieve value quickly with your Auth0 enterprise subscription. The 6 steps in the onboarding program set both you and Auth0 up for success.
+    </p>
+  </li>
+  <li>
+    <i class="icon icon-budicon-715"></i><a href="/appliance/private-cloud-requirements">Information Requirements for Setting Up the Appliance in Auth0's Private Cloud</a>
+    <p>
+      If you have decided to purchase an Appliance that is hosted in a dedicated area of Auth0's cloud, Auth0 will set up the Appliance on your behalf. To do so, Auth0 asks that you provide the following information:
```
Comment (mpaktiti) [inline articles/appliance/index.html]: @nkohari This probably needs to go: `To do so, Auth0 asks that you provide the following information:`

**Pull request merged**